roles/create_vms/templates/create_vm.sh.j2

```diff
@@ -1,5 +1,10 @@
 #! /bin/bash
 
+# Copy VARS.fd file related to non secure boot mode if that's the case
```

Comment:
this comment should be inside the if block

Reply:
ack